sdk/keyvault/azsecrets/README.md

```diff
@@ -345,52 +342,42 @@ func main() {
 
 ### Error Handling
 
-All I/O operations return a non-nil `error` when they fail. These errors include their motivating HTTP response, if any:
-```golang
-resp, err := client.GetSecret(context.TODO(), "mySecretName", nil)
+All methods which send HTTP requests return an `error` when these requests fail. For deeper debugging, you can capture the raw response from Key Vault with `runtime.WithCaptureResponse()`:
```

Comment:
For the error case, we should be converting to an `*azcore.ResponseError` which contains the raw HTTP response.